laravel-directory-sync-app/resources/views/users.blade.php

```diff
@@ -1,41 +1,59 @@
+            <div>
+            <img src="{{ URL::to('/images/workos-logo-with-text.png') }}"alt="workos logo">
+            </div>
+
         </div>
         <div>
-            <a href="https://workos.com/" target="_blank"><button class='button'>WorkOS</button></a>
-            <a href="https://workos.com/docs" target="_blank"><button class='button'>Documentation</button></a>
-            <a href="https://workos.com/docs/reference" target="_blank"><button class='button'>API Reference</button></a>
-            <a href="https://workos.com/blog" target="_blank"><button class='button'>Blog</button></a>
-            
+            <a href="https://workos.com/docs" target="_blank"><button class='button nav-item'>Documentation</button></a>
+            <a href="https://workos.com/docs/reference" target="_blank"><button class='button nav-item'>API
+                    Reference</button></a>
+            <a href="https://workos.com/blog" target="_blank"><button
+                    class='button nav-item blog-nav-button'>Blog</button></a>
+            <a href="https://workos.com/" target="_blank"><button class='button button-outline'>WorkOS</button></a>
         </div>
     </div>
+    <div class='flex'>
+        <div class="logged_in_div_left">
+            <div class="title-text">
+                <h1>Your app,</h1>
+                <h2 class="home-hero-gradient">Enterprise Ready</h2>
+            </div>
+            <div class="title-subtext">
+                <p>Start selling to enterprise customers with just a few lines of code.</p>
+                <p>Implement features like single sign-on in minutes instead of months.</p>
+            </div>
+            <div class="flex success-buttons">
+                <a href="https://workos.com/signup" target="_blank"><button class='button'>Get Started</button></a>
+                <a href="mailto:sales@workos.com?subject=WorkOS Sales Inquiry" target="_blank"><button
+                        class='button button-outline sales-button'>Contact
+                        Sales</button></a>
+            </div>
+        </div>
     <div class="logged_in_div_right">
-        <a href="{{ url('/') }}"><button class='button back'> ❮ Back </button></a>
-
         <div class="flex_column">
         @if ($users)
             <h2>Users</h2>
+            <div class="text_box">
+
                 <ul style="list-style: none;">
                 @foreach ($users as $user)
-                <div class="text_box">
-                        <h4 class = "center">{{$user->id}}<h4>
+                        <h4 class = "center">{{$user -> firstName}} {{$user -> lastName}} - {{$user->id}}<h4>
                         <li class = "center">
-                            <a href="{{ url('directory/'.$directoryId . '/users/'.$user->id) }}"><button class='button'>Users</button></a>
+                            <a href="{{ url('directory/'.$directoryId . '/users/'.$user->id) }}"><button class='button'>View User</button></a>
```

Comment:
This list of users looks a little clunky to me. Could we either adjust the position of the button so it's to the right of the user's name, or get rid of the buttons and make the user's name a link that goes to the user page for that user? 
![CleanShot 2022-08-26 at 14 29 34](https://user-images.githubusercontent.com/47123183/186986145-a59e2ee9-b9a0-43c9-b681-33a3a41f1522.png)

Reply:
done